src/pmdas/statsd/src/pmda-callbacks.c

```diff
@@ -366,7 +367,6 @@ map_metric(char* key, struct metric* item, void* pmda) {
     if (item->meta->pcp_instance_change_requested == 1) {
         update_pcp_metric_instance_domain(key, item, (pmdaExt*)pmda);
     }
-    data->pcp_metric_count += 1;
```

Comment:
This incremented total mapped metric count by 1, for each rebuild of PMNS, meaning that it also counted metrics that already had their counterpart pmdaMetric structures in pmdaRehash(pmda, data->pcp_metrics, data->pcp_metric_count);'s data->pcp_metrics.

Reply:
Good find @Erbenos !  I'm out today but will try it tomorrow on the reproducer system we have here.

Reply:
@Erbenos this is looking good, the reproducer setup has been running for 10+ minutes now without failure 🥇 💯

Reply:
Only other thing I noticed here is that the statsd PMDA is pretty chatty - I'm not sure how normal a load this is but after 10-15 minutes the PMDA log file is already ~13MB large (over a day or two this is going to be causing some folks ENOSPC problems).  So we probably should reduce that ...I suggest moving some of the detail to under a debug flag, and just keep this one info-level line on reconfiguration:
```
[Thu Sep 19 02:18:24] pmdastatsd(1006338) Info: statsd: pmdastatsd: 920 metrics and 3 instance domains after reload
```

Here's what I'm seeing currently...
```
ls -lh /var/log/pcp/pmcd/statsd.log
-rw-r--r--. 1 root root 13M Sep 19 02:18 /var/log/pcp/pmcd/statsd.log
```

Reply:
Depends on the verbosity set, can always be tweaked if necessary if a specific verbosity level feels like it has unjustifiable footprint

Reply:
Yeah, based on what I saw yesterday the defaults need tweaking - default should be more like one line of output on reconfiguration and verbose output is opt-in for when deeper diagnostics / debugging is required.